internal/cmd/dockerbuild/main.go

```diff
@@ -64,18 +64,19 @@ func run(basedir string, dockerOrg string, args []string) error {
 			return err
 		}
 		var pluginKey string
+		const bazelPluginKey = "bazel"
```

Comment:
This made a huge difference - having two bazel builds running at the same time caused major slowdowns. Here's an example after this change where you both see the caching and more reasonable build times: https://github.com/bufbuild/plugins/actions/runs/6117484022/job/16604171425